contentpacks/khanacademy.py

```diff
@@ -425,7 +425,7 @@ def retrieve_exercise_dict(lang=None, force=False) -> str:
 
 
 @cache_file
-def download_and_clean_kalite_data(url, path) -> str:
+def download_and_clean_kalite_data(url, path, lang) -> str:
```

Comment:
Is this arg going to get passed? The `cache_file` decorator only passes `(url, path, **kwargs)`.